static/app/components/charts/components/tooltip.tsx

```diff
@@ -26,11 +26,15 @@ export function defaultFormatAxisLabel(
     return value;
   }
 
+  const formatOptions = {local: !utc};
+
+  const timeFormat = showTimeInTooltip
+    ? getTimeFormat({displaySeconds: addSecondsToTimeFormat})
+    : '';
+
   if (!bucketSize) {
-    const format = `MMM D, YYYY ${
-      showTimeInTooltip ? getTimeFormat({displaySeconds: addSecondsToTimeFormat}) : ''
-    }`.trim();
-    return getFormattedDate(value, format, {local: !utc});
+    const format = `MMM D, YYYY ${timeFormat} z`.trim();
```

Comment:
I don't think this trim() is doing anything right now.

Reply:
will circle back and clean this up